src/base/utils/net.cpp

```diff
@@ -38,5 +39,52 @@ namespace Utils
         {
             return !QHostAddress(ip).isNull();
         }
+
+        Subnet parseSubnet(const QString &subnetStr, bool *ok) {
+            const Subnet invalid = qMakePair(QHostAddress(), -1);
+            const Subnet subnet = QHostAddress::parseSubnet(subnetStr);
+            *ok = subnet != invalid;
```

Comment:
`if (ok) *ok = (subnet != invalid);`

Reply:
Will implement this change. But for my own info, what's the purpose of checking `ok` before assigning it?

Reply:
It can be `nullptr` if user don't want to know status.